client/webserver/site/src/js/markets.ts

```diff
@@ -875,6 +888,7 @@ export default class MarketsPage extends BasePage {
     // candlesticks
     this.candleDur = fiveMinBinKey
     this.loadCandles()
+    this.candleDurationSelected(oneHrBinKey)
```

Comment:
This sort of defeats the purpose of remembering the user's selection if you're going to force it to one-hour bins every time they load a new market. We typically would do something like
```Typescript
 this.candleDur = State.fetchLocal(State.lastCandleDurationLK) || oneHrBinKey
```
in the constructor.



Also, you're calling `loadCandles` and `candleDurationSelected` also calls `loadCandles`

Reply:
> This sort of defeats the purpose of remembering the user's selection if you're going to force it to one-hour bins every time they load a new market

Uhmm, yeah. I thought of this but maybe I didn't fully understand why we were supposed to set the 1h bin as default.

Reply:
Yeah, the snipped I posted was not considering the local storage bit.

But the idea was a simple way to make the default be 1hr, but still always fetch and cache the 5min candles for the high/low reasons you mentioned.  The simplest way to do that is just to do loadCandles twice in a row.  I'm open to any solution, but we just need the 1hr candles as the default in 0.6.1 one way or another.

Reply:
> Uhmm, yeah. I thought of this but maybe I didn't fully understand why we were supposed to set the 1h bin as default.

Default means when they haven't chosen a bin already.

The reason 5m is no longer good for the default is because the market looks flat with practical trading volumes.

Reply:
The `if (this.candleDur !== fiveMinBinKey) {` snippet that's been added in `setHighLow` should take care of extra loading for older servers, but will not incur an extraneous request for new servers.

Reply:
I think I'm miscommunicating "default".  There's no need to store or persist it because it's just in the code.  If the user has never selected a bin size themselves, then we use the default bin size of 1h.

Reply:
>If the user has never selected a bin size themselves, then we use the default bin size of 1h.

Yeah. I believe my recent commits resolve our concerns correctly and https://github.com/decred/dcrdex/pull/2284#discussion_r1171530003.